apps/frontend/src/components/Tag/Tag.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import vector from '../../assets/icons/vector.svg';
```

Comment:
Also, rename this vector to something meaningful. Like `cross` or something

Reply:
will rename it to cross